src/zjs_buffer.c

```diff
@@ -414,12 +416,12 @@ static jerry_value_t zjs_buffer(const jerry_value_t function_obj,
         jerry_size_t size = 0;
         char *str = zjs_alloc_from_jstring(argv[0], &size);
         if (!str) {
-            return zjs_error("zjs_buffer: could not allocate string");
+            return zjs_error("could not allocate string");
```

Comment:
This one you've got to look at the actual code, not just the diff... :) It's a separate else section... but thanks for thinking about the mallocs.